apps/pie-microsite/src/_11ty/shortcodes/tokensTable/index.js

```diff
@@ -19,17 +19,22 @@ const {
  * @param {string} tokenType - the type of token i.e. color, spacing, radius
  * @returns {string} the SCSS variable name
  */
-const createScssTokenName = (tokenKey, tokenType) => `$${tokenType}-${tokenKey}`;
+const createScssTokenName = (tokenKey, tokenType, path = '') => {
+    const prefixExcludes = [path === 'radius.alias' ? tokenTypes.RADIUS : []];
```

Comment:
I think what would be cleaner and more extensible here would be to branch the logic based on token type and have separate handler for picky tokens like radius, and a default for others. Similar to what we do the for create example element function :)